master/internal/rm/multirm/multirm.go

```diff
@@ -415,6 +415,16 @@ func (m *MultiRMRouter) VerifyNamespaceExists(namespaceName string,
 	return nil
 }
 
+func (m *MultiRMRouter) DeleteNamespace(namespaceName string) error {
+	err := m.fanOutRMCommand(func(rm rm.ResourceManager) error {
+		return rm.DeleteNamespace(namespaceName)
+	})
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
little nit, just my style preference -- you could condense all this code and just return line 419 -- lines 422-425 don't add more information than the err value return on 419

Reply:
It's my style preference too 🙈😂, good catch!